### PR TITLE
Scaffold top portion of balances page

### PR DESCRIPTION
**Pull request by dgca**

### CODE REVIEW

File: packages/mobile-app/.eslintrc.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ module.exports = {
   plugins: ["prettier"],
   rules: {
     "prettier/prettier": "error",
+    "import/no-unresolved": "off",
```
Comment (dgca) [inline packages/mobile-app/.eslintrc.js:7]: I can't figure out how to set up path aliases while also making ESLint happy

Comment (cgjohn) [reply packages/mobile-app/.eslintrc.js:7]: Only suggestion would be to add a comment for why we have it

File: packages/tackle-box/lib/components/Text/Text.tsx [hunk 2]
```diff
@@ -6,40 +6,33 @@ type Sizes = "4xl" | "3xl" | "2xl" | "xl" | "lg" | "md" | "sm" | "xs";
 const styles = css.create({
   base: {
     fontFamily: "Favorit",
+    textAlign: "center",
```
Comment (cgjohn) [inline packages/tackle-box/lib/components/Text/Text.tsx:9]: Does this not get overwritten by the `textAlign` prop that we default to `left`?

File: packages/tackle-box/lib/svg.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite-plugin-svgr/client" />
```
Comment (cgjohn) [inline packages/tackle-box/lib/svg.d.ts:1]: Is this what fixed the import error message?

**Review by cgjohn — APPROVED** [review]
two _tiny_ thoughts but lgtm!

File: packages/mobile-app/.eslintrc.js (hunk 1 above)
Comment (dgca) [reply packages/mobile-app/.eslintrc.js]: Just fixed it hah

File: packages/tackle-box/lib/components/Text/Text.tsx (hunk 2 above)
Comment (dgca) [reply packages/tackle-box/lib/components/Text/Text.tsx:9]: Good catch 👍

File: packages/tackle-box/lib/svg.d.ts (hunk 3 above)
Comment (dgca) [reply packages/tackle-box/lib/svg.d.ts:1]: Nah, this was to get SVGs working. The thing that fixed the import error message is here: https://github.com/iron-fish/mobile-wallet/pull/72/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357R7

**Pull request merged**